src/containers/Comments/Comment.js

```diff
@@ -61,16 +63,15 @@ export const Comment = (props) => {
   const getCommentDeleteModalItems = (commentInstance) => {
     return [
       {
-        // space is to center the text
-        title: '   This comment will be deleted.\nThis action cannot be undone.',
+        title: [t('This comment will be deleted.'), t('This action cannot be undone.')],
```

Comment:
removed the whitespaces as it doesn't work for translated strings anymore (see comment above)

Reply:
Cool! This is much better and on android it's still centered. :)